### ISSUE TITLE
Header being duplicated when there is no comment on the root key

### ISSUE THREAD

**Issue by EverNife** [Carleslc/Simple-YAML#60]
When there is no comment on the root-key, the header will be in both the options().header and in the roots.comment()

This will cause a **duplication** of the header on the next save.

<details>
  <summary>Example!</summary>
  
![image](https://user-images.githubusercontent.com/11357801/167241006-6e75e3e9-f840-4a56-a8f1-58a43c9b1497.png)

</details>

-------------------------------

How to Reproduce:

**Under the YamlCommentsFormatExample.java**

**1 -** Remove the comment on RootKey https://github.com/Carleslc/Simple-YAML/blob/master/Simple-Yaml/src/test/java/org/simpleyaml/examples/YamlCommentsFormatExample.java#L45

**2 -** Add this to the end of the test

```
        YamlFile yamlFile2 = new YamlFile(yamlFile.getConfigurationFile());
        yamlFile2.loadWithComments();
        yamlFile2.save("examples/test-comments-format-2.yml");
```

FileOutput:

![image](https://user-images.githubusercontent.com/11357801/167241122-5d4e78d4-0687-4ee5-816d-5c0804d2345c.png)

**Comment by Carleslc** [comment]
A blank line is a valid comment for the library (that's how you add new blank lines as comments), that's why the header comment is assigned to the first root key.

As a workaround until the next release, if you don't want any comment in the root key, you can remove the comment of the first root key:

```java
YamlFile yamlFile2 = new YamlFile(yamlFile.getConfigurationFile());
yamlFile2.loadWithComments();

// Remove the duplicated header
yamlFile2.setComment("[0]", null);

System.out.println("yamlFile2.getHeader()");
System.out.println(yamlFile2.getHeader());

System.out.println("\nyamlFile2.getComment(\"key-1\")");
System.out.println(yamlFile2.getComment("key-1"));

yamlFile2.save("examples/test-comments-format-2.yml");
```

```
yamlFile2.getHeader()
######################
##  HEADER COMMENT  ##
######################

yamlFile2.getComment("key-1")
null
```

```yaml
######################
##  HEADER COMMENT  ##
######################

key-1: This is a root key
```

**Comment by EverNife** [comment]
Ty, as workround for now i am comparing the header and the root-key comment and erasing the root-key-comment when they até the same.

**Comment by Carleslc** [comment]
Fixed in [1.8.1](https://github.com/Carleslc/Simple-YAML/releases/tag/1.8.1)
